lti_consumer/data.py

```diff
@@ -24,9 +26,19 @@ class Lti1p3ProctoringLaunchData:
         assessment message to after it has completed the proctoring setup and verification. This attribute is required
         if the message_type attribute of the Lti1p3LaunchData instance is "LtiStartProctoring". It is optional and
         unused otherwise.
+    * assessment_control_url (optional): The Platform URL that the Tool will send assessment control messages to.
+    * assessment_control_actions (optional): A list of assessment control actions supported by the platform.
     """
     attempt_number = field()
     start_assessment_url = field(default=None)
+    assessment_control_url = field(default=None)
```

Comment:
Should we somehow validate that if one of these is supplied that the other is also necessary? They're both required, and I have done some validation of the data class previously. We do have a [validate_lti_1p3_launch_data](https://github.com/openedx/xblock-lti-consumer/blob/3eaf8aa7ce0f3bd601bf318db48d7f8db50445e8/lti_consumer/api.py#L247) function we could add to. I think that's easier than validating when initializing the `Lti1p3LaunchData` class, unless you know how to do that.